examples/data/demo/webactions.json

```diff
@@ -1 +1 @@
erceptor","misk.web.interceptors.RequestLoggingInterceptor","misk.web.interceptors.MetricsInterceptor","misk.web.interceptors.TracingInterceptor","misk.web.exceptions.ExceptionHandlingInterceptor","misk.web.interceptors.MarshallerInterceptor","misk.web.interceptors.WideOpenDevelopmentInterceptor"],"parameterTypes":[],"pathPattern":"/_tab/url-create-short/{path:.*}","requestMediaTypes":["*/*"],"responseMediaType":"*/*","returnType":"misk.web.Response<misk.web.ResponseBody>"},{"applicationInterceptors":["misk.security.authz.AccessInterceptor"],"dispatchMechanism":"POST","function":"fun com.squareup.cashurlshortener.actions.CreateShortUrlWebAction.createShortUrl(com.squareup.cashurlshortener.actions.CreateShortUrlWebAction.Request): com.squareup.cashurlshortener.actions.CreateShortUrlWebAction.Response","functionAnnotations":["@misk.web.Post(pathPattern=/create)","@misk.web.RequestContentType(value=[application/json;charset=utf-8])","@misk.web.ResponseContentType(value=application/json;charset=utf-8)","@com.squareup.cashurlshortener.service.CashUrlshortenerOperatorAccess()"],"name":"CreateShortUrlWebAction","networkInterceptors":["misk.web.interceptors.InternalErrorInterceptorFactory$Companion$INTERCEPTOR$1","misk.web.interceptors.RequestLogContextInterceptor","misk.web.interceptors.RequestLoggingInterceptor","misk.web.interceptors.MetricsInterceptor","misk.web.interceptors.TracingInterceptor","misk.web.exceptions.ExceptionHandlingInterceptor","misk.web.interceptors.MarshallerInterceptor","misk.web.interceptors.WideOpenDevelopmentInterceptor"],"parameterTypes":["com.squareup.cashurlshortener.actions.CreateShortUrlWebAction.Request"],"pathPattern":"/create","requestMediaTypes":["application/json;charset=utf-8"],"responseMediaType":"application/json;charset=utf-8","returnType":"com.squareup.cashurlshortener.actions.CreateShortUrlWebAction.Response"},{"applicationInterceptors":["misk.security.authz.AccessInterceptor"],"dispatchMechanism":"GET","function":"fun com.squareup.cashurlshortener.actions.ShortUrlWebAction.follow(kotlin.String): misk.web.Response<misk.web.ResponseBody>","functionAnnotations":["@misk.web.Get(pathPattern=/{token:[^/_]+})","@misk.web.ResponseContentType(value=application/json;charset=utf-8)","@com.squareup.cashurlshortener.service.CashUrlshortenerOperatorAccess()"],"name":"ShortUrlWebAction","networkInterceptors":["misk.web.interceptors.InternalErrorInterceptorFactory$Companion$INTERCEPTOR$1","misk.web.interceptors.RequestLogContextInterceptor","misk.web.interceptors.RequestLoggingInterceptor","misk.web.interceptors.MetricsInterceptor","misk.web.interceptors.TracingInterceptor","misk.web.exceptions.ExceptionHandlingInterceptor","misk.web.interceptors.MarshallerInterceptor","misk.web.interceptors.WideOpenDevelopmentInterceptor"],"parameterTypes":["kotlin.String"],"pathPattern":"/{token:[^/_]+}","requestMediaTypes":["*/*"],"responseMediaType":"application/json;charset=utf-8","returnType":"misk.web.Response<misk.web.ResponseBody>"}]}
```

Comment:
could we expand this out? Pretty unreadable

Reply:
This is generated JSON from the `webActions.ts` in src, checked in so it's accessible by URL.

Reply:
Ah gotcha, thanks